bindings/nodejs/lib/aio.js

```diff
@@ -23,16 +23,23 @@ var soletta = require( 'bindings' )( 'soletta' ),
 exports.open = function( init ) {
 	return new Promise( function( fulfill, reject ) {
 		var precision = 12;
+		var aio;
 
 		if (init.precision)
 			precision = init.precision;
 
 		if (init.name && init.name != "") {
```

Comment:
On 25-04-2016 14:24, Bruno Bottazzini wrote:
> In bindings/nodejs/lib/aio.js
> <https://github.com/solettaproject/soletta/pull/1929#discussion_r60952408>:
>
>> @@ -23,16 +23,23 @@ var soletta = require( 'bindings' )( 'soletta' ),
>>  exports.open = function( init ) {
>>  	return new Promise( function( fulfill, reject ) {
>>  		var precision = 12;
>> +		var aio;
>>
>>  		if (init.precision)
>>  			precision = init.precision;
>>
>>  		if (init.name && init.name != "") {
>
> Use !== instead of !=

Hmmm... I didn't touch this line in my patch... But I may do that...

>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1929/files/30d66fd8a71c3298abd559b43eee4edec8322eec#r60952408>
>